stable/chronograf/templates/NOTES.txt

```diff
@@ -0,0 +1,28 @@
+Chronograf can be accessed via port 80 on the following DNS name from within your cluster:
+
+- http://{{ template "fullname" . }}.{{ .Release.Namespace }}
+
+You can easily connect to the remote instance from your browser. Forward the webserver port to localhost:8888
+
+- kubectl port-forward --namespace {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l app={{ template "fullname" . }} -o jsonpath='{ .items[0].metadata.name }') 8888:80
```

Comment:
You need to forward local 8888 to 8888 (ie the container not service port). 

`kubectl port-forward --namespace {{ .Release.Namespace }} $(kubectl get pods --namespace {{ .Release.Namespace }} -l app={{ template "fullname" . }} -o jsonpath='{ .items[0].metadata.name }') 8888`